src/main/java/uk/gov/companieshouse/controller/AppealController.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.companieshouse.controller;
+
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AppealController {
```

Comment:
That does not deserve to be here :) Similar to my comment around a service that was not needed at the time, here we do not know whether we need one controller or many controllers. We do not know how we will split responsibilities between these controllers. We do not know what names they will have.

Reply:
Ok, makes sense - removing this now